hc-sr04-range-sensor/Makefile

```diff
@@ -92,6 +107,98 @@ $(ELF1): $(SRC1) $(HEADERS) | $(OUT)
 clean:
 	$(RM) -fr $(ELF0) $(ELF1) $(OUT)
 
+stop_pru0:
+	@if [ $(ARCHT) = $(ARM) ]; then\
```

Comment:
If I do "make stop_pru0" on x86 it will do nothing but will declare success. That is misleading. I would instead suggest you to do the following:

```
HOST_CHECK_CMD:=[ `uname -m` = armv7l ] || { echo "ERROR: Host is not a BeagleBone board! Cannot install PRU firmware. Bailing out."; false; }


stop_pru0:
	@$(HOST_CHECK_CMD)
	@echo "Do some work here."
	@echo "Next command."
```

IMHO it is more clear to read, and will fail if unable to service the requested action.

Reply:
@deebot , I cannot open the link you have provided.The snippet I provided works - I tested it on beaglebone and x86_64 host.